### PR TITLE
feat: add  tenant info into usage_schema tables

### PR DESCRIPTION
**Pull request by nakahaa**
# Which issue does this PR close?

Closes #.

# Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: query_server/query/src/metadata/usage_schema_provider/mod.rs [hunk 1]
```diff
@@ -104,12 +104,7 @@ pub fn create_usage_schema_view_table(
...
 
-    let cols = builder
-        .schema()
-        .fields()
-        .iter()
-        .filter(|f| f.name().ne("tenant"))
-        .map(|f| col(f.name()));
+    let cols = builder.schema().fields().iter().map(|f| col(f.name()));
 
     let logical_plan = builder.clone().project(cols)?.build()?;
```
Comment (yukkit) [inline query_server/query/src/metadata/usage_schema_provider/mod.rs:109]:
```suggestion
    let logical_plan = builder.build()?;
```

Projection can be removed directly

File: main/src/http/http_service.rs [hunk 2]
```diff
@@ -95,6 +96,7 @@ impl HttpService {
         dbms: DBMSRef,
         coord: CoordinatorRef,
         addr: SocketAddr,
+        host: String,
```
Comment (yukkit) [inline main/src/http/http_service.rs:99]: Can SocketAddr be used directly? What is the reason why it cannot be used directly？

Comment (nakahaa) [reply main/src/http/http_service.rs:99]: the default socketaddr is 0.0.0.0,

File: query_server/query/src/metadata/usage_schema_provider/mod.rs (hunk 1 above)
Comment (nakahaa) [reply query_server/query/src/metadata/usage_schema_provider/mod.rs]: done

**Review by roseboy-liu — APPROVED** [review]

**Comment by roseboy-liu** [comment]
useless commit

**Pull request closed without merging**